CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     message(FATAL_ERROR "Need Clang 9.0 or newer to build XGBoost")
   endif()
 endif()
+if(APPLE)
+  set(CMAKE_MACOSX_DEPLOYMENT_TARGET 10.15)
```

Comment:
Is it possible to use this as an independent command line argument in CI instead of hardcoding it into the source code?

Reply:
Sure, let me create a PR.